packages/utilities/psammead-styles/src/fonts.js

```diff
@@ -1,14 +1,188 @@
-// BBC News Article fonts
-const fontFamilyBase = ', Helvetica, Arial, sans-serif';
-
-export const FF_NEWS_SANS_LIT = `ReithSansNewsLight${fontFamilyBase}`;
-export const FF_NEWS_SANS_REG = `ReithSansNewsRegular${fontFamilyBase}`;
-export const FF_NEWS_SANS_REG_ITAL = `ReithSansNewsRegularItalic${fontFamilyBase}`;
-export const FF_NEWS_SANS_MDM = `ReithSansNewsMedium${fontFamilyBase}`;
-export const FF_NEWS_SANS_BLD = `ReithSansNewsBold${fontFamilyBase}`;
-
-export const FF_NEWS_SERIF_LIT = `ReithSerifNewsLight${fontFamilyBase}`;
-export const FF_NEWS_SERIF_REG = `ReithSerifNewsRegular${fontFamilyBase}`;
-export const FF_NEWS_SERIF_MDM = `ReithSerifNewsMedium${fontFamilyBase}`;
-export const FF_NEWS_SERIF_MDM_ITAL = `ReithSerifNewsMediumItalic${fontFamilyBase}`;
-export const FF_NEWS_SERIF_BLD = `ReithSerifNewsBold${fontFamilyBase}`;
+const baseFontUrl = 'https://gel.files.bbci.co.uk/r2.302/';
+
+// Serif
+export const T_REITH_SERIF_REGULAR = `
+  @font-face {
+    font-family: "ReithSerif";
+    src: url("${baseFontUrl}BBCReithSerif_W_Rg.woff2") format("woff2"), url("${baseFontUrl}BBCReithSerif_W_Rg.woff") format("woff");
+    font-display: optional;
+  }`;
+
+export const T_REITH_SERIF_ITALIC = `
+  @font-face {
+    font-family: "ReithSerif";
+    src: url("${baseFontUrl}BBCReithSerif_W_It.woff2") format("woff2"), url("${baseFontUrl}BBCReithSerif_W_It.woff") format("woff");
+    font-style: italic;
+    font-display: optional;
+  }`;
+
+export const T_REITH_SERIF_BOLD = `
+  @font-face {
+    font-family: "ReithSerif";
+    src: url("${baseFontUrl}BBCReithSerif_W_Bd.woff2") format("woff2"), url("${baseFontUrl}BBCReithSerif_W_Bd.woff") format("woff");
+    font-weight: bold;
```

Comment:
Can I be a pain and ask we move to using `700` instead of bold please? more consistant with the medium and light IMO

Reply:
Done.